src/Boundaries/Sediments/simple_multi_G.jl

```diff
@@ -170,56 +174,56 @@ sediment_fields(model::SimpleMultiG) = (C_slow = model.fields.C_slow,
+        sediment_tendencies.N_ref[i, j, 1] = sediment.refactory_fraction * nitrogen_deposition
 
         # efflux/influx
-        O₂  = tracers.O₂[i, j, 1]
-        NO₃ = tracers.NO₃[i, j, 1]
-        NH₄ = tracers.NH₄[i, j, 1]
+        O₂  = tracers.O₂[i, j, k]
+        NO₃ = tracers.NO₃[i, j, k]
+        NH₄ = tracers.NH₄[i, j, k]
+        
+        A, B, C, D, E, F = sediment.nitrate_oxidation_params
 
-        pₙᵢₜ = exp(sediment.nitrate_oxidation_params.A +
-                   sediment.nitrate_oxidation_params.B * log(Cᵐⁱⁿ * day) * log(O₂) +
-                   sediment.nitrate_oxidation_params.C * log(Cᵐⁱⁿ * day) ^ 2 +
-                   sediment.nitrate_oxidation_params.D * log(k) * log(NH₄) +
-                   sediment.nitrate_oxidation_params.E * log(Cᵐⁱⁿ * day) +
-                   sediment.nitrate_oxidation_params.F * log(Cᵐⁱⁿ * day) * log(NH₄)) / (Nᵐⁱⁿ * day)
+        pₙᵢₜ = exp(A +
+                  B * log(Cᵐⁱⁿ * day) * log(O₂) +
+                  C * log(Cᵐⁱⁿ * day) ^ 2 +
+                  D * log(reactivity) * log(NH₄) +
+                  E * log(Cᵐⁱⁿ * day) +
+                  F * log(Cᵐⁱⁿ * day) * log(NH₄)) / (Nᵐⁱⁿ * day)
 
+        #=
         pᵈᵉⁿⁱᵗ = exp(sediment.denitrification_params.A +
                      sediment.denitrification_params.B * log(Cᵐⁱⁿ * day) +
                      sediment.denitrification_params.C * log(NO₃) ^ 2 +
                      sediment.denitrification_params.D * log(Cᵐⁱⁿ * day) ^ 2 +
-                     sediment.denitrification_params.E * log(k) ^ 2 +
-                     sediment.denitrification_params.F * log(O₂) * log(k)) / (Cᵐⁱⁿ * day)
-
-        pₐₙₒₓ = exp(sediment.anoxic_params.A +
-                    sediment.anoxic_params.B * log(Cᵐⁱⁿ * day) +
-                    sediment.anoxic_params.C * log(Cᵐⁱⁿ * day) ^ 2 +
-                    sediment.anoxic_params.D * log(k) +
-                    sediment.anoxic_params.E * log(O₂) * log(k) +
-                    sediment.anoxic_params.F * log(NO₃) ^ 2) / (Cᵐⁱⁿ * day)
-
-        if isnan(pₐₙₒₓ)
-            println("$(Cᵐⁱⁿ), $(k), $(O₂), $(NO₃)")
-            error("Sediment anoxia has caused model failure")
-        end
-
-        pₛₒₗᵢ = sediment.solid_dep_params.A * (sediment.solid_dep_params.C * sediment.solid_dep_params.depth ^ sediment.solid_dep_params.D) ^ sediment.solid_dep_params.B
-
-        Δz = grid.Δzᵃᵃᶜ[1]
-
-        timestepper.Gⁿ.NH₄[i, j, 1] += Nᵐⁱⁿ * (1 - pₙᵢₜ) / Δz
-        timestepper.Gⁿ.NO₃[i, j, 1] += Nᵐⁱⁿ * pₙᵢₜ / Δz
-        timestepper.Gⁿ.DIC[i, j, 1] += Cᵐⁱⁿ / Δz
-        timestepper.Gⁿ.O₂[i, j, 1]  -= max(0, (1 - pᵈᵉⁿⁱᵗ - pₐₙₒₓ * pₛₒₗᵢ) * Cᵐⁱⁿ / Δz) # this seems dodge but this model doesn't cope with anoxia properly
+                     sediment.denitrification_params.E * log(reactivity) ^ 2 +
+                     sediment.denitrification_params.F * log(O₂) * log(reactivity)) / (Cᵐⁱⁿ * day)
+        =#
```

Comment:
shall we delete these commented code?

Reply:
I guess so, might be better todo in #138 since I've made a lot of changes to the sediment there